### PR TITLE
Added suggested dates

### PR DESCRIPTION
**Pull request by ZombieAlienRobot**
## Description of changes
Added suggested start and end dates to chapters and update the validator for them.

  
## How has this been tested?
Please describe the test strategy you followed.

- [X] automated unit test
- [X] automated integration test
- [ ] automated acceptance test
- [ ] manual, exploratory test
    
## Checklist before requesting a review
- [X] My code follows the [coding guidelines](https://github.com/IT-REX-Platform/wiki/blob/main/dev-manuals/backend/coding-guidelines.md) of this project
- [X] I have performed a self-review of my code
- [X] I have commented my code, particularly in hard-to-understand areas
- [X] My code fulfills all acceptance criteria
- [X] I have added tests that prove my fix is effective or that my feature works
- [X] New and existing unit tests pass locally with my changes
- [X] The test coverage (line and branch) is reasonably high, especially on Service classes and other classes with complex logic
- [X] I have made corresponding changes to the documentation
- [X] I have added explanation of architectural decision and rationales to [wiki/adr](https://github.com/IT-REX-Platform/wiki/tree/main/adr)
- [X] I have updated the changes in the ticket description

## Checklist for reviewer
- [ ] The code works and does not throw errors
- [ ] The code is easy to understand and there are no confusing parts
- [ ] The code follows the [coding guidelines](https://github.com/IT-REX-Platform/wiki/blob/main/dev-manuals/backend/coding-guidelines.md) of this project
- [ ] The code change accomplishes what it is supposed to do
- [ ] I cannot think of any use case in which the code does not behave as intended
- [ ] The added and existing tests reasonably cover the code change
- [ ] I cannot think of any test cases, input or edge cases that should be tested in addition
- [ ] Description of the change is included in the documentation

### CODE REVIEW

File: src/main/resources/graphql/chapter.graphqls [hunk 1]
```diff
@@ -60,6 +70,16 @@ input CreateChapterInput {
     """
     endDate: DateTime!
     """
+    Suggested Start date to start the chapter, ISO 8601 format.
+    Must be after Start Date and before the End dates.
+    """
+    suggestedStartDate: DateTime
```
Comment (myluki2000) [inline src/main/resources/graphql/chapter.graphqls:76]: Why are these nullable but the ones in the `Chapter` type are not?

**Review by myluki2000 — CHANGES_REQUESTED** [review]

**Review by myluki2000 — APPROVED** [review]
